packages/modal/README.md

```diff
@@ -87,14 +87,36 @@ This code will render:
 
   > Props to be passed to the [Card](https://www.npmjs.com/package/@crave/farmblocks-card) component that wraps the modal.
 
-- **closeProps** (_Object_)
+- **closeButtonProps** (_Object_)
 
-  > Props to be passed to the [Link](https://www.npmjs.com/package/@crave/farmblocks-link) component used for the close icon.
+  > Props to be passed to the [Button](https://www.npmjs.com/package/@crave/farmblocks-button) component used for the close button.
 
 - **zIndex** (_Number_) = `1500`
 
   > Value for the `z-index` [CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/z-index).
 
+- **header** (_Node_)
+
+  > Node for the header section, if **showCloseButton** is _true_ will add an offset space in case of header is not provided
+
+- **headerProps** (_Object_)
+
+  > Properties passed to the header section
+
+- **footer** (_Node_)
+
+  > Node for the footer section
+
+- **footerProps** (_Object_)
+
+  > Properties passed to the footer section
+
+### Helper component
+
+- **ModalTitle** (_Component_)
+
+  > Component to use on **header** prop, that opniated styles your title [Text](https://www.npmjs.com/package/@crave/farmblocks-text)
```

Comment:
I don't think this should be in the `API` section.
It could be along with `DialogModal`, maybe renaming the section to `Extra Components`.

And what do you mean by `opniated`?